include/bpf_helper_defs.h

```diff
@@ -129,4 +129,18 @@ EBPF_HELPER(uint64_t, bpf_ktime_get_ns, ());
 EBPF_HELPER(int, bpf_csum_diff, (void* from, int from_size, void* to, int to_size, int seed));
 #ifndef __doxygen
 #define bpf_csum_diff ((bpf_csum_diff_t)BPF_FUNC_csum_diff)
+#endif
+
+/**
+ * @brief Copy data into the ring buffer map.
+ *
+ * @param[in,out] map Pointer to ring buffer map.
+ * @param[in] data Data to copy into ring buffer map.
+ * @param[in] size Length of data.
+ * @param[in] flags Flags indicating if notification for new data availability be sent.
```

Comment:
Can't parse grammar.  Should this be something like
```suggestion
 * @param[in] flags Flags indicating if notification for new data availability should be sent.
```
?